src/traces/bar/plot.js

```diff
@@ -124,8 +124,10 @@ module.exports = function plot(gd, plotinfo, cdbar) {
                     // append bar path and text
                     var bar = d3.select(this);
 
-                    bar.append('path').attr('d',
-                        'M' + x0 + ',' + y0 + 'V' + y1 + 'H' + x1 + 'V' + y0 + 'Z');
+                    bar.append('path')
```

Comment:
Non- :no_entry: comment 

----------

What about bar text labels?

![peek 2017-07-19 11-39](https://user-images.githubusercontent.com/6675409/28375957-e99437ce-6c76-11e7-9473-506482af0188.gif)

they probably would benefit from the same [tricks](https://github.com/plotly/plotly.js/blob/97d08c8a126e9dd84e95acc8c8edc655eb95ed28/src/plots/cartesian/dragbox.js#L763) scatter text nodes get on scroll